README.md

```diff
@@ -1,14 +1,20 @@
+# Sticky Notes
+Sticky Notes replaces GNote as the default application for taking notes. Similar to GNote, Sticky is developed in GTK3, supports HiDPI and integrates very well in the desktop environment. In addition to the features which were present in GNote, Sticky also lets you place the notes on the desktop and quickly access them from the icon tray.
```

Comment:
This is only true for Linux Mint, and since this app is meant to be distro-agnostic, Mint-specific language should probably not be in the app description. Also, Sticky wasn't actually written to be a replacement for gnote either. It was meant to be an easy-to-use note-taking app that was accessible and familiar to the average user. While it has many of the same features as gnote, the app serves a different purpose than gnote does, and isn't a direct replacement. Gnote is still a better choice for certain users. Instead, this description should focus on what the app is.

This section might read something like "Sticky is a note-taking app for the linux desktop, that simulates traditional "sticky note" style stationary on your desktop. Some of it's features include basic text formatting (bold, italics, etc), spell-checking, a tray icon for controlling note visibility, manual and automatic backups, and a manager to organize your notes into groups. Sticky is written in Python, and uses the Gtk3 toolkit."

Reply:
thanks for the clarification, just updated like this